services/director-v2/src/simcore_service_director_v2/core/settings.py

```diff
@@ -168,7 +171,7 @@ class DynamicSidecarEgressSettings(BaseCustomSettings):
         "envoyproxy/envoy:v1.25-latest",
         description="envoy image to use",
     )
-    DYNAMIC_SIDECAR_ENVOY_LOG_LEVEL: EnvoyLogLevel = Field(
+    DYNAMIC_SIDECAR_ENVOY_LOG_LEVEL: EnvoyLogLevel = Field(  # type: ignore
```

Comment:
why the ignore here?
also ```default=EnvoyLogLevel.ERROR``` so that pylance is happy

Reply:
here it complains about the `auto` used in the enum